sdks/java/extensions/schemaio-expansion-service/build.gradle

```diff
@@ -31,11 +31,15 @@ applyJavaNature(
 
 dependencies {
     compile project(path: ":sdks:java:expansion-service")
+    permitUnusedDeclared project(path: ":sdks:java:expansion-service")
     compile project(":sdks:java:io:jdbc")
+    permitUnusedDeclared project(":sdks:java:io:jdbc")
     compile library.java.postgres
+    permitUnusedDeclared library.java.postgres
+    compile project(path: ":model:pipeline", configuration: "shadow")
+    compile project(path: ":sdks:java:core", configuration: "shadow")
+    compile library.java.vendored_grpc_1_26_0
+    compile library.java.vendored_guava_26_0_jre
     testCompile library.java.junit
-    testCompile library.java.powermock_mockito
     testCompile library.java.mockito_core
-    // TODO(BEAM-10632): remove this dependency
-    testCompile "org.checkerframework:checker-qual:3.10.0"
```

Comment:
This introduced by @kennknowles back in Oct. Since then we've moved the checker-qual dependency into BeamModulePlugin. Checking if it's ok to remove it from here.